code/modules/tooltip/tooltip.dm

```diff
@@ -81,15 +81,16 @@ Notes:
 
 /datum/tooltip/proc/hide()
 	if (src.queueHide)
-		spawn(1)
-			winshow(src.owner, src.control, 0)
+		addtimer(CALLBACK(src, .proc/do_hide), 1)
```

Comment:
it's not a param it's a delay of 1 for whatever reason 
it's gooncode or somethign ask @Tacolizard but I didn't want to touch it.